src/Illuminate/Database/Eloquent/Collection.php

```diff
@@ -139,7 +139,7 @@ public function loadMissing($relations)
     /**
      * Load a relationship path if it is not already eager loaded.
      *
-     * @param  \Illuminate\Database\Eloquent\Collection  $models
+     * @param  static $models
```

Comment:
Please don't change this. It is a stylistic choice to use the full name in the phpdoc.

Reply:
this is actually not the same. static is wrong here